<!DOCTYPE html>
<html lang="en"><script>
    (function() {
    class RuffleMimeType {
        constructor(a, b, c) {
            this.type = a, this.description = b, this.suffixes = c
        }
    }
    class RuffleMimeTypeArray {
        constructor(a) {
            this.__mimetypes = [], this.__named_mimetypes = {};
            for (let b of a) this.install(b)
        }
        install(a) {
            let b = this.__mimetypes.length;
            this.__mimetypes.push(a), this.__named_mimetypes[a.type] = a, this[a.type] = a, this[b] = a
        }
        item(a) {
            return this.__mimetypes[a]
        }
        namedItem(a) {
            return this.__named_mimetypes[a]
        }
        get length() {
            return this.__mimetypes.length
        }
    }
    class RufflePlugin extends RuffleMimeTypeArray {
        constructor(a, b, c, d) {
            super(d), this.name = a, this.description = b, this.filename = c
        }
        install(a) {
            a.enabledPlugin || (a.enabledPlugin = this), super.install(a)
        }
    }
    class RufflePluginArray {
        constructor(a) {
            this.__plugins = [], this.__named_plugins = {};
            for (let b of a) this.install(b)
        }
        install(a) {
            let b = this.__plugins.length;
            this.__plugins.push(a), this.__named_plugins[a.name] = a, this[a.name] = a, this[b] = a
        }
        item(a) {
            return this.__plugins[a]
        }
        namedItem(a) {
            return this.__named_plugins[a]
        }
        get length() {
            return this.__plugins.length
        }
    }
    const FLASH_PLUGIN = new RufflePlugin("Shockwave Flash", "Shockwave Flash 32.0 r0", "ruffle.js", [new RuffleMimeType("application/futuresplash", "Shockwave Flash", "spl"), new RuffleMimeType("application/x-shockwave-flash", "Shockwave Flash", "swf"), new RuffleMimeType("application/x-shockwave-flash2-preview", "Shockwave Flash", "swf"), new RuffleMimeType("application/vnd.adobe.flash-movie", "Shockwave Flash", "swf")]);

    function install_plugin(a) {
        navigator.plugins.install || Object.defineProperty(navigator, "plugins", {
            value: new RufflePluginArray(navigator.plugins),
            writable: !1
        }), navigator.plugins.install(a), 0 < a.length && !navigator.mimeTypes.install && Object.defineProperty(navigator, "mimeTypes", {
            value: new RuffleMimeTypeArray(navigator.mimeTypes),
            writable: !1
        });
        for (var b = 0; b < a.length; b += 1) navigator.mimeTypes.install(a[b])
    }
    install_plugin(FLASH_PLUGIN);
})();
</script><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>KEGG API Manual</title>
<link rel="stylesheet" href="KEGG%20API%20Manual_files/kegg25.css">
<style>
th {
    font-weight: normal;
    text-align: left;
    text-decoration: underline;
}
</style>
<style data-id="immersive-translate-input-injected-css">.immersive-translate-input {
  position: absolute;
  top: 0;
  right: 0;
  left: 0;
  bottom: 0;
  z-index: 2147483647;
  display: flex;
  justify-content: center;
  align-items: center;
}
.immersive-translate-attach-loading::after {
  content: " ";

  --loading-color: #f78fb6;
  width: 6px;
  height: 6px;
  border-radius: 50%;
  display: block;
  margin: 12px auto;
  position: relative;
  color: white;
  left: -100px;
  box-sizing: border-box;
  animation: immersiveTranslateShadowRolling 1.5s linear infinite;

  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-2000%, -50%);
  z-index: 100;
}

.immersive-translate-loading-spinner {
  vertical-align: middle !important;
  width: 10px !important;
  height: 10px !important;
  display: inline-block !important;
  margin: 0 4px !important;
  border: 2px rgba(221, 244, 255, 0.6) solid !important;
  border-top: 2px rgba(0, 0, 0, 0.375) solid !important;
  border-left: 2px rgba(0, 0, 0, 0.375) solid !important;
  border-radius: 50% !important;
  padding: 0 !important;
  -webkit-animation: immersive-translate-loading-animation 0.6s infinite linear !important;
  animation: immersive-translate-loading-animation 0.6s infinite linear !important;
}

@-webkit-keyframes immersive-translate-loading-animation {
  from {
    -webkit-transform: rotate(0deg);
  }

  to {
    -webkit-transform: rotate(359deg);
  }
}

@keyframes immersive-translate-loading-animation {
  from {
    transform: rotate(0deg);
  }

  to {
    transform: rotate(359deg);
  }
}


.immersive-translate-input-loading {
  --loading-color: #f78fb6;
  width: 6px;
  height: 6px;
  border-radius: 50%;
  display: block;
  margin: 12px auto;
  position: relative;
  color: white;
  left: -100px;
  box-sizing: border-box;
  animation: immersiveTranslateShadowRolling 1.5s linear infinite;
}

@keyframes immersiveTranslateShadowRolling {
  0% {
    box-shadow: 0px 0 rgba(255, 255, 255, 0), 0px 0 rgba(255, 255, 255, 0), 0px 0 rgba(255, 255, 255, 0), 0px 0 rgba(255, 255, 255, 0);
  }

  12% {
    box-shadow: 100px 0 var(--loading-color), 0px 0 rgba(255, 255, 255, 0), 0px 0 rgba(255, 255, 255, 0), 0px 0 rgba(255, 255, 255, 0);
  }

  25% {
    box-shadow: 110px 0 var(--loading-color), 100px 0 var(--loading-color), 0px 0 rgba(255, 255, 255, 0), 0px 0 rgba(255, 255, 255, 0);
  }

  36% {
    box-shadow: 120px 0 var(--loading-color), 110px 0 var(--loading-color), 100px 0 var(--loading-color), 0px 0 rgba(255, 255, 255, 0);
  }

  50% {
    box-shadow: 130px 0 var(--loading-color), 120px 0 var(--loading-color), 110px 0 var(--loading-color), 100px 0 var(--loading-color);
  }

  62% {
    box-shadow: 200px 0 rgba(255, 255, 255, 0), 130px 0 var(--loading-color), 120px 0 var(--loading-color), 110px 0 var(--loading-color);
  }

  75% {
    box-shadow: 200px 0 rgba(255, 255, 255, 0), 200px 0 rgba(255, 255, 255, 0), 130px 0 var(--loading-color), 120px 0 var(--loading-color);
  }

  87% {
    box-shadow: 200px 0 rgba(255, 255, 255, 0), 200px 0 rgba(255, 255, 255, 0), 200px 0 rgba(255, 255, 255, 0), 130px 0 var(--loading-color);
  }

  100% {
    box-shadow: 200px 0 rgba(255, 255, 255, 0), 200px 0 rgba(255, 255, 255, 0), 200px 0 rgba(255, 255, 255, 0), 200px 0 rgba(255, 255, 255, 0);
  }
}


.immersive-translate-search-recomend {
  border: 1px solid #dadce0;
  border-radius: 8px;
  padding: 16px;
  margin-bottom: 16px;
  position: relative;
  font-size: 16px;
}

.immersive-translate-search-enhancement-en-title {
  color: #4d5156;
}

/* dark */
@media (prefers-color-scheme: dark) {
  .immersive-translate-search-recomend {
    border: 1px solid #3c4043;
  }

  .immersive-translate-close-action svg {
    fill: #bdc1c6;
  }

  .immersive-translate-search-enhancement-en-title {
    color: #bdc1c6;
  }
}


.immersive-translate-search-settings {
  position: absolute;
  top: 16px;
  right: 16px;
  cursor: pointer;
}

.immersive-translate-search-recomend::before {
  /* content: " "; */
  /* width: 20px; */
  /* height: 20px; */
  /* top: 16px; */
  /* position: absolute; */
  /* background: center / contain url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAMAAABEpIrGAAAAxlBMVEUAAADpTInqTIjpSofnSIfqS4nfS4XqS4nqTIjsTYnrTInqTIroS4jvQIDqTIn////+/v7rSYjpTIn8/v7uaZzrTIr9/f3wfansWJL88/b85e73qc39+/v3xNnylrvrVI/98fb62Obva5/8+fr76vH4y9zpSIj74e353Oj1ocTzm77xhK/veKbtYpjsXJTqU47oTInxjrXyh7L99fj40eH2ttH1udD3sc31ssz1rMnykLXucqPtbqD85e/1xdn2u9DzqcXrUY6FaJb8AAAADnRSTlMA34BgIM8Q37/fz7+/EGOHcVQAAAGhSURBVDjLhZPncuowEEZFTW7bXVU7xsYYTO/p7bb3f6lICIOYJOT4h7/VnFmvrBFjrF3/CR/SajBHswafctG0Qg3O8O0Xa8BZ6uw7eLjqr30SofCDVSkemMinfL1ecy20r5ygR5zz3ArcAqJExPTPKhDENEmS30Q9+yo4lEQkqVTiIEAHCT10xWERRdH0Bq0aCOPZNDV3s0xaYce1lHEoDHU8wEh3qRJypNcTAeKUIjgKMeGLDoRCLVLTVf+Ownj8Kk6H9HM6QXPgYjQSB0F00EJEu10ILQrs/QeP77BSSr0MzLOyuJJQbnUoOOIUI/A8EeJk9E4YUHUWiRyTVKGgQUB8/3e/NpdGlfI+FMQyWsCBWyz4A/ZyHXyiiz0Ne5aGZssoxRmcChw8/EFKQ5JwwkUo3FRT5yXS7q+Y/rHDZmFktzpGMvO+5QofA4FPpEmGw+EWRCFvnaof7Zhe8NuYSLR0xErKLThUSs8gnODh87ssy6438yzbLzxl012HS19vfCf3CNhnbWOL1eEsDda+gDPUvri8tSZzNFrwIZf1NmNvqC1I/t8j7nYAAAAASUVORK5CYII='); */
}

.immersive-translate-search-title {}

.immersive-translate-search-title-wrapper {}

.immersive-translate-search-time {
  font-size: 12px;
  margin: 4px 0 24px;
  color: #70757a;
}

.immersive-translate-expand-items {
  display: none;
}

.immersive-translate-search-more {
  margin-top: 16px;
  font-size: 14px;
}

.immersive-translate-modal {
  display: none;
  position: fixed;
  z-index: 2147483647;
  left: 0;
  top: 0;
  width: 100%;
  height: 100%;
  overflow: auto;
  background-color: rgb(0, 0, 0);
  background-color: rgba(0, 0, 0, 0.4);
  font-size: 15px;
}

.immersive-translate-modal-content {
  background-color: #fefefe;
  margin: 10% auto;
  padding: 40px 24px 24px;
  border: 1px solid #888;
  border-radius: 10px;
  width: 80%;
  max-width: 270px;
  font-family: system-ui, -apple-system, "Segoe UI", "Roboto", "Ubuntu",
    "Cantarell", "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji",
    "Segoe UI Symbol", "Noto Color Emoji";
  position: relative
}

.immersive-translate-modal .immersive-translate-modal-content-in-input {
  max-width: 500px;
}
.immersive-translate-modal-content-in-input .immersive-translate-modal-body {
  text-align: left;
  max-height: unset;
}

.immersive-translate-modal-title {
  text-align: center;
  font-size: 16px;
  font-weight: 700;
  color: #333333;
}

.immersive-translate-modal-body {
  text-align: center;
  font-size: 14px;
  font-weight: 400;
  color: #333333;
  word-break: break-all;
  margin-top: 24px;
}

@media screen and (max-width: 768px) {
  .immersive-translate-modal-body {
    max-height: 250px;
    overflow-y: auto;
  }
}

.immersive-translate-close {
  color: #666666;
  position: absolute;
  right: 16px;
  top: 16px;
  font-size: 20px;
  font-weight: bold;
}

.immersive-translate-close:hover,
.immersive-translate-close:focus {
  color: black;
  text-decoration: none;
  cursor: pointer;
}

.immersive-translate-modal-footer {
  display: flex;
  justify-content: center;
  flex-wrap: wrap;
  margin-top: 24px;
}

.immersive-translate-btn {
  width: fit-content;
  color: #fff;
  background-color: #ea4c89;
  border: none;
  font-size: 16px;
  margin: 0 8px;
  padding: 9px 30px;
  border-radius: 5px;
  display: flex;
  align-items: center;
  justify-content: center;
  cursor: pointer;
  transition: background-color 0.3s ease;
}

.immersive-translate-btn:hover {
  background-color: #f082ac;
}

.immersive-translate-cancel-btn {
  /* gray color */
  background-color: rgb(89, 107, 120);
}


.immersive-translate-cancel-btn:hover {
  background-color: hsl(205, 20%, 32%);
}

.immersive-translate-action-btn {
  background-color: transparent;
  color: #EA4C89;
  border: 1px solid #EA4C89
}

.immersive-translate-btn svg {
  margin-right: 5px;
}

.immersive-translate-link {
  cursor: pointer;
  user-select: none;
  -webkit-user-drag: none;
  text-decoration: none;
  color: #007bff;
  -webkit-tap-highlight-color: rgba(0, 0, 0, .1);
}

.immersive-translate-primary-link {
  cursor: pointer;
  user-select: none;
  -webkit-user-drag: none;
  text-decoration: none;
  color: #ea4c89;
  -webkit-tap-highlight-color: rgba(0, 0, 0, .1);
}

.immersive-translate-modal input[type="radio"] {
  margin: 0 6px;
  cursor: pointer;
}

.immersive-translate-modal label {
  cursor: pointer;
}

.immersive-translate-close-action {
  position: absolute;
  top: 2px;
  right: 0px;
  cursor: pointer;
}</style></head>

<body>

<div id="content" style="margin-top:20px; margin-left:40px;">

<h2 class="txt3"><a href="https://www.kegg.jp/kegg/rest/"><img src="KEGG%20API%20Manual_files/kegg3.gif" class="middle" alt="kegg3 logo"></a> &nbsp;KEGG API Manual</h2>

<a href="https://www.kegg.jp/kegg/rest/">KEGG API</a> is a REST-style Application Programming Interface to the KEGG database resource.

<br>
<hr class="frame3">

<h4 class="txt3">General form</h4>

<div class="indent">

<b class="txt3">URL form</b>
<pre class="block">https://rest.kegg.jp/&lt;operation&gt;/&lt;argument&gt;[/&lt;argument2[/&lt;argument3&gt; ...]]

&lt;operation&gt; = <a href="#info">info</a> | <a href="#list">list</a> | <a href="#find">find</a> | <a href="#get">get</a> | <a href="#conv">conv</a> | <a href="#link">link</a> | <a href="#ddi">ddi</a></pre>

<b class="txt3">Database name</b>
<pre class="block">&lt;database&gt; = <i>KEGG databases (Table 1) and Outside databases integrated in KEGG (Table 2)</i>

           = pathway | brite | module | ko | &lt;org&gt; | vg | vp | ag | genome | compound |
             glycan | reaction | rclass | enzyme | network | variant | disease |
             drug | dgroup | disease_ja | drug_ja | dgroup_ja | compound_ja |
             genes | ligand | kegg | &lt;outside_db&gt;

&lt;org&gt; = <i>KEGG organism code</i>

&lt;outside_db&gt; = pubmed | ncbi-geneid | ncbi-proteinid | uniprot | pubchem | chebi |
               atc | jtc | ndc | yj</pre>

<b class="txt3">Database entry identifier</b>
<pre class="block">&lt;dbentry&gt; = &lt;kid&gt; | &lt;org&gt;:&lt;gene&gt; | &lt;database&gt;:&lt;entry&gt;

&lt;kid&gt; = <i>KEGG identifier</i>
&lt;gene&gt; = <i>Gene entry name or accession</i>
&lt;entry&gt; = <i>Database entry name or accession</i>

&lt;dbentries&gt; = &lt;dbentry&gt;<sub>1</sub>[+&lt;dbentry&gt;<sub>2</sub>...]
</pre>

</div>

<h4 class="txt3">Naming conventions</h4>

<div class="indent">

KEGG is an integrated database consisting of sixteen databases 
(including four Japanese versions) shown in Table 1. Except for "genes",
 "enzyme" and "variant", each database entry is identified by the KEGG 
identifier &lt;kid&gt; consisting of a database-dependent prefix 
followed by a five-digit number (see <a href="https://www.kegg.jp/kegg/kegg3.html">KEGG Objects</a>
 for more details), such as K number, C number and D numbers as 
identifiers of "ko", "compound" and "drug" databases, respectively.
<br><br>
A "genes" entry is identified by
<div class="indent">
&lt;org&gt;:&lt;gene&gt;
</div>
where &lt;org&gt; is the three- or four-letter KEGG organism code or the
 T number genome identifier and &lt;gene&gt; is the gene identifier, 
usually NCBI GeneID or INSDC Locus_tag (see <a href="https://www.kegg.jp/kegg/genes.html">KEGG GENES</a>).
<br><br>
An "enzyme" entry, a "variant" entry or an entry of any outside database is identified by
<div class="indent">
&lt;database&gt;:&lt;entry&gt;
</div>
where &lt;database&gt; is the database name or its abbreviation defined 
in Tables 1-2 and &lt;entry&gt; is the entry name or the accession 
number given by the database.

<div class="indent">
<p>Table 1. KEGG databases</p>
<table class="frame3"><tbody><tr><td>
<table>
<tbody><tr>
  <td>DB name</td>
  <td>Abbrev</td>
  <td>Content</td>
  <td>Web page</td>
  <td>kid prefix</td>
</tr>
<tr>
  <td colspan="5"><hr class="frame3"></td>
</tr>
<tr>
  <td>pathway</td>
  <td>path</td>
  <td>KEGG pathway maps</td>
  <td><a href="https://www.kegg.jp/kegg/pathway.html">KEGG PATHWAY</a></td>
  <td>map, ko, ec, rn, &lt;org&gt;</td>
</tr>
<tr>
  <td>brite</td>
  <td>br</td>
  <td>BRITE functional hierarchies</td>
  <td><a href="https://www.kegg.jp/kegg/brite.html">KEGG BRITE</a></td>
  <td>br, jp, ko, &lt;org&gt;</td>
</tr>
<tr>
  <td>module</td>
  <td>md</td>
  <td>KEGG modules</td>
  <td><a href="https://www.kegg.jp/kegg/module.html">KEGG MODULE</a></td>
  <td>M, &lt;org&gt;_M</td>
</tr>
<tr>
  <td>orthology</td>
  <td>ko</td>
  <td>KO functional orthologs</td>
  <td><a href="https://www.kegg.jp/kegg/ko.html">KEGG ORTHOLOGY</a></td>
  <td>K</td>
</tr>
<tr>
  <td>genes</td>
  <td>&lt;org&gt;<br>vg<br>vp<br>ag</td>
  <td>Genes in KEGG organisms<br>Genes in viruses category<br>Mature peptides in viruses<br>Genes in addendum category</td>
  <td><a href="https://www.kegg.jp/kegg/genes.html">KEGG GENES</a></td>
  <td>-</td>
</tr>
<tr>
  <td>genome</td>
  <td>gn</td>
  <td>KEGG organisms</td>
  <td><a href="https://www.kegg.jp/kegg/genome.html">KEGG GENOME</a></td>
  <td>T</td>
</tr>
<tr>
  <td>compound</td>
  <td>cpd</td>
  <td>Small molecules</td>
  <td><a href="https://www.kegg.jp/kegg/compound/">KEGG COMPOUND</a></td>
  <td>C</td>
</tr>
<tr>
  <td>glycan</td>
  <td>gl</td>
  <td>Glycans</td>
  <td><a href="https://www.kegg.jp/kegg/glycan/">KEGG GLYCAN</a></td>
  <td>G</td>
</tr>
<tr>
  <td>reaction</td>
  <td>rn</td>
  <td>Biochemical reactions</td>
  <td><a href="https://www.kegg.jp/kegg/reaction/">KEGG REACTION</a></td>
  <td>R</td>
</tr>
<tr>
  <td>rclass</td>
  <td>rc</td>
  <td>Reaction class</td>
  <td></td>
  <td>RC</td>
</tr>
<tr>
  <td>enzyme</td>
  <td>ec</td>
  <td>Enzyme nomenclature</td>
  <td><a href="https://www.kegg.jp/kegg/annotation/enzyme.html">KEGG ENZYME</a></td>
  <td>-</td>
</tr>
<tr>
  <td>network</td>
  <td>ne</td>
  <td>Network elements</td>
  <td><a href="https://www.kegg.jp/kegg/network.html">KEGG NETWORK</a></td>
  <td>N</td>
</tr>
<tr>
  <td>variant</td>
  <td>hsa_var</td>
  <td>Human gene variants</td>
  <td></td>
  <td>-</td>
</tr>
<tr>
  <td>disease</td>
  <td>ds</td>
  <td>Human diseases</td>
  <td><a href="https://www.kegg.jp/kegg/disease/">KEGG DISEASE</a></td>
  <td>H</td>
</tr>
<tr>
  <td>drug</td>
  <td>dr</td>
  <td>Drugs</td>
  <td><a href="https://www.kegg.jp/kegg/drug/">KEGG DRUG</a></td>
  <td>D</td>
</tr>
<tr>
  <td>dgroup</td>
  <td>dg</td>
  <td>Drug groups</td>
  <td></td>
  <td>DG</td>
</tr>
<tr>
  <td colspan="5"><hr class="frame3"></td>
</tr>
<tr>
  <td>disease_ja</td>
  <td>ds_ja</td>
  <td>H number</td>
  <td colspan="2"><a href="https://www.kegg.jp/kegg/disease/disease_ja.html">KEGG DISEASE</a> (in Japanese)</td>
</tr>
<tr>
  <td>drug_ja</td>
  <td>dr_ja</td>
  <td>D number</td>
  <td colspan="2"><a href="https://www.kegg.jp/kegg/drug/drug_ja.html">KEGG DRUG</a> (in Japanese)</td>
</tr>
<tr>
  <td>dgroup_ja</td>
  <td>dg_ja</td>
  <td>DG number</td>
  <td colspan="2"></td>
</tr>
<tr>
  <td>compound_ja</td>
  <td>cpd_ja</td>
  <td>C number</td>
  <td colspan="2"></td>
</tr>
</tbody></table>
</td></tr></tbody></table>
<ul style="margin-right:60px;">
<li>"genes" is a composite database consisting of KEGG organisms with 
three- or four-letter &lt;org&gt; codes, and viruses (vg, vp) and 
addendum (ag) categories (see <a href="https://www.kegg.jp/kegg/genes.html">KEGG GENES</a>).</li>
<li>"pathway", "brite" and "module" consist of manually created 
reference datasets and computationally generated organism-specific 
datasets (see <a href="https://www.kegg.jp/kegg/kegg3a.html">KEGG Pathway Maps</a>).</li>
<li>"kegg" stands for the collection of all databases shown above.</li>
<li>"ligand" stands for the collection of chemical databases: compound, glycan, reaction and enzyme.</li>
</ul>
</div>

<br>
In addition to these databases, the NCBI PubMed database shown in Table 2
 is tightly integrated in KEGG. Table 2 also contains databases used 
only for ID conversion (conv operation) or links (link operation).

<div class="indent">
<p>Table 2. Outside databases integrated in KEGG</p>
<table class="frame3"><tbody><tr><td>
<table>
<tbody><tr>
  <td>DB name</td>
  <td>Abbrev</td>
  <td>Entry ID</td>
  <td>Web page</td>
  <td>Remark</td>
</tr>
<tr>
  <td colspan="5"><hr class="frame3"></td>
</tr>
<tr>
  <td>pubmed</td>
  <td>pmid</td>
  <td>PubMed ID</td>
  <td><a href="https://pubmed.ncbi.nlm.nih.gov/">NCBI PubMed</a></td>
  <td></td>
</tr>
<tr>
  <td>ncbi-geneid</td>
  <td></td>
  <td>Gene ID</td>
  <td><a href="https://www.ncbi.nlm.nih.gov/gene/">NCBI Gene</a></td>
  <td>conv only</td>
</tr>
<tr>
  <td>ncbi-proteinid</td>
  <td></td>
  <td>Protein ID</td>
  <td><a href="https://www.ncbi.nlm.nih.gov/protein/">NCBI Protein</a></td>
  <td>conv only</td>
</tr>
<tr>
  <td>uniprot</td>
  <td>up</td>
  <td>UniProt Accession</td>
  <td><a href="https://www.uniprot.org/">UniProt</a></td>
  <td>conv only</td>
</tr>
<tr>
  <td>pubchem</td>
  <td></td>
  <td>PubChem SID</td>
  <td><a href="https://www.ncbi.nlm.nih.gov/pubchem/">NCBI PubChem</a></td>
  <td>conv only</td>
</tr>
<tr>
  <td>chebi</td>
  <td></td>
  <td>ChEBI ID</td>
  <td><a href="https://www.ebi.ac.uk/chebi/">ChEBI</a></td>
  <td>conv only</td>
</tr>
<tr>
  <td></td>
  <td>atc</td>
  <td>7-letter ATC code</td>
  <td><a href="https://www.kegg.jp/brite/br08303">ATC classification</a></td>
  <td>link only</td>
</tr>
<tr>
  <td></td>
  <td>jtc</td>
  <td>Therapeutic category code</td>
  <td><a href="https://www.kegg.jp/brite/br08301">Therapeutic category in Japan</a></td>
  <td>link only</td>
</tr>
<tr>
  <td></td>
  <td>ndc</td>
  <td>National Drug Code</td>
  <td>Drug products in the USA</td>
  <td>ddi only</td>
</tr>
<tr>
  <td></td>
  <td>yj</td>
  <td>YJ code</td>
  <td>Drug products in Japan</td>
  <td>ddi only</td>
</tr>
</tbody></table>
</td></tr></tbody></table>
</div>

</div>

<h4 class="txt3">Output</h4>

<div class="indent">

<b class="txt3">Output format</b>
<div class="indent2">
The output of all operations is in a text format:
<ul>
<li>tab-delimited text returned from list, find, conv and link</li>
<li><a href="https://www.kegg.jp/kegg/docs/dbentry.html">flat file database format</a> returned from get</li>
<li>text message returned from info</li>
</ul>
</div>

<b class="txt3">Status code</b>
<div class="indent2">
<p>The HTTP status code can be used to check if the operation was successful.</p>
<table class="frame3"><tbody><tr><td>
<table>
<tbody><tr><th>Code &nbsp;</th><th>Meaning</th></tr>
<tr><td>200</td><td>Success</td></tr>
<tr><td>400</td><td>Bad request (syntax error, wrong database name, etc.)</td></tr>
<tr><td>404</td><td>Not found (e.g., requesting amino acid sequence for RNA)</td></tr>
</tbody></table>
</td></tr></tbody></table>
</div>

</div>

<br>
<hr class="frame3">

<h2 class="txt3">KEGG API Operations</h2>

<h4 class="txt3" id="info">INFO</h4>

<div class="indent">

<b class="txt3">Name</b>
<div class="indent2">
info – display database release information and linked db information
</div>

<b class="txt3">URL form</b>
<pre class="block">https://rest.kegg.jp/info/&lt;database&gt;

&lt;database&gt; = kegg | pathway | brite | module | ko | genes | &lt;org&gt; | vg | vp | ag |
             genome | ligand | compound | glycan | reaction | rclass | enzyme |
             network | variant | disease | drug | dgroup</pre>

<b class="txt3">Description</b>
<div class="indent2">
This operation displays the database release information with statistics
 for the databases shown in Table 1. Except for kegg, genes and ligand, 
this operation also displays the list of linked databases that can be 
used in the link operation.
</div>

<b class="txt3">Examples</b>
<div class="indent2">
<table>
<tbody><tr>
  <td><a href="https://rest.kegg.jp/info/kegg">/info/kegg</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>displays the current statistics of the KEGG database</td>
</tr>
<tr class="top">
  <td><a href="https://rest.kegg.jp/info/pathway">/info/pathway</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>displays the number of pathway entries including both the reference and organism-specific pathways</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/info/hsa">/info/hsa</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>displays the number of gene entries for the KEGG organism <i>Homo sapiens</i></td>
</tr>
</tbody></table>
</div>

<b class="txt3">See also</b>
<div class="indent2">
<a href="https://www.kegg.jp/kegg/docs/statistics.html">Current statistics</a> in the KEGG website.
</div>

</div>

<br>
<hr class="frame3">

<h4 class="txt3" id="list">LIST</h4>

<div class="indent">

<b class="txt3">Name</b>
<div class="indent2">
list – obtain a list of entry identifiers and associated names
</div>

<b class="txt3">URL form</b>
<pre class="block">https://rest.kegg.jp/list/&lt;database&gt;

&lt;database&gt; = pathway | brite | module | ko | &lt;org&gt; | vg | vp | ag | genome | compound |
             glycan | reaction | rclass | enzyme | network | variant | disease |
             drug | dgroup | organism</pre>

<pre class="block">https://rest.kegg.jp/list/pathway/&lt;org&gt;</pre>

<pre class="block">https://rest.kegg.jp/list/brite/&lt;option&gt;

&lt;option&gt; = br | jp | ko | &lt;org&gt;</pre>

<pre class="block">https://rest.kegg.jp/list/&lt;dbentries&gt;

&lt;dbentries&gt; = <i>Entries of the following</i> &lt;database&gt;
&lt;database&gt; = pathway | brite | module | ko | &lt;org&gt; | vg | vp | ag | genome | compound |
             glycan | reaction | rclass | enzyme | network | variant | disease |
             drug | dgroup</pre>

<b class="txt3">Description</b>
<div class="indent2">
This operation can be used to obtain a list of all entries in each 
database. The database names shown in Tables 1 and 2, excluding the 
composite database names of genes, ligand and kegg, may be given. The 
special database name "organism" is allowed only in this operation, 
which may be used to obtain a list of KEGG organisms with the three- or 
four-letter organism codes.
<br><br>
When the organism code is known, the second form can be used to obtain a list of organism-specific pathways.
<br><br>
The third form is a similar option for brite hierarchies.
<br><br>
The fourth form may be used to obtain a list of definitions for a given 
set of database entry identifiers. The maximum number of identifiers 
that can be given is 10.
</div>

<b class="txt3">Examples</b>
<div class="indent2">
<table>
<tbody><tr>
  <td><a href="https://rest.kegg.jp/list/pathway">/list/pathway</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>returns the list of reference pathways</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/list/pathway/hsa">/list/pathway/hsa</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>returns the list of human pathways</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/list/organism">/list/organism</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>returns the list of KEGG organisms with taxonomic classification</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/list/hsa">/list/hsa</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>returns the entire list of human genes with gene types and chromosomal positions</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/list/T01001">/list/T01001</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>same as above</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/list/hsa:10458+ece:Z5100">/list/hsa:10458+ece:Z5100</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>returns the list of a human gene and an E.coli O157 gene</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/list/C01290+G00092">/list/C01290+G00092</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>returns the list of a compound entry and a glycan entry</td>
</tr>
</tbody></table>
</div>

</div>

<br>
<hr class="frame3">

<h4 class="txt3" id="find">FIND</h4>

<div class="indent">

<b class="txt3">Name</b>
<div class="indent2">
find – find entries with matching query keyword or other query data
</div>

<b class="txt3">URL form</b>
<pre class="block">https://rest.kegg.jp/find/&lt;database&gt;/&lt;query&gt;

&lt;database&gt; = pathway | brite | module | ko | genes | &lt;org&gt; | vg | vp | ag | genome |
             ligand | compound | glycan | reaction | rclass | enzyme | network |
             variant | disease | drug | dgroup</pre>

<pre class="block">https://rest.kegg.jp/find/&lt;database&gt;/&lt;query&gt;/&lt;option&gt;

&lt;database&gt; = compound | drug
&lt;option&gt; = formula | exact_mass | mol_weight | nop</pre>

<b class="txt3">Description</b>
<div class="indent2">
This is a search operation. The first form searches entry identifier and associated fields shown below for matching keywords.
<br><br>
<div class="indent">
<table class="frame3">
<tbody><tr><th>Database</th><th>Text search fields (see <a href="https://www.kegg.jp/kegg/rest/dbentry.html">flat file format</a>)</th></tr>
<tr><td>pathway</td><td>ENTRY and NAME</td></tr>
<tr><td>brite</td><td>ENTRY and NAME</td></tr>
<tr><td>module</td><td>ENTRY and NAME</td></tr>
<tr><td>ko</td><td>ENTRY, SYMBOL and NAME</td></tr>
<tr><td>genes (&lt;org&gt;, vg, vp, ag)</td><td>ENTRY, SYMBOL, NAME and KO</td></tr>
<tr><td>genome</td><td>ENTRY, ORG_CODE and NAME</td></tr>
<tr><td>compound</td><td>ENTRY and NAME</td></tr>
<tr><td>glycan</td><td>ENTRY, NAME and COMPOSITION</td></tr>
<tr><td>reaction</td><td>ENTRY, NAME and DEFINITION</td></tr>
<tr><td>rclass</td><td>ENTRY and DEFINITION</td></tr>
<tr><td>enzyme</td><td>ENTRY and NAME</td></tr>
<tr><td>network</td><td>ENTRY and NAME</td></tr>
<tr><td>variant</td><td>ENTRY and NAME</td></tr>
<tr><td>disease</td><td>ENTRY and NAME</td></tr>
<tr><td>drug</td><td>ENTRY and NAME</td></tr>
<tr><td>dgroup</td><td>ENTRY and NAME</td></tr>
</tbody></table>
</div>
<br>
In the second form the chemical formula search is a partial match 
irrespective of the order of atoms given. The exact mass (or molecular 
weight) is checked by rounding off to the same decimal place as the 
query data. A range of values may also be specified with the minus(-) 
sign.
</div>

<b class="txt3">Examples</b>
<div class="indent2">
<table>
<tbody><tr>
  <td><a href="https://rest.kegg.jp/find/genes/shiga+toxin">/find/genes/shiga+toxin</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>for keywords "shiga" and "toxin" (use nop option to disable this processing)</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/find/genes/shiga%20toxin">/find/genes/"shiga&nbsp;toxin"</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>for keywords "shiga toxin"</td>
</tr>
</tbody></table>
<table>
<tbody><tr>
  <td><a href="https://rest.kegg.jp/find/compound/C7H10O5/formula">/find/compound/C7H10O5/formula</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>for chemical formula "C7H10O5"</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/find/compound/O5C7/formula">/find/compound/O5C7/formula</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>for chemical formula containing "O5" and "C7"</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/find/compound/174.05/exact_mass">/find/compound/174.05/exact_mass</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>for 174.045 =&lt; exact mass &lt; 174.055</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/find/compound/300-310/mol_weight">/find/compound/300-310/mol_weight</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>for 300 =&lt; molecular weight =&lt; 310</td>
</tr>
</tbody></table>
</div>

<b class="txt3">See also</b>
<div class="indent2">
DBGET search interface such as in the <a href="https://www.kegg.jp/kegg/kegg2.html">KEGG Table of Contents</a> page.
</div>

</div>

<br>
<hr class="frame3">

<h4 class="txt3" id="get">GET</h4>

<div class="indent">

<b class="txt3">Name</b>
<div class="indent2">
get – retrieve given database entries
</div>

<b class="txt3">URL form</b>
<pre class="block">https://rest.kegg.jp/get/&lt;dbentries&gt;[/&lt;option&gt;]

&lt;dbentries&gt; = <i>KEGG database entries of the following</i> &lt;database&gt;
&lt;database&gt; = pathway | brite | module | ko | &lt;org&gt; | vg | vp | ag | genome | compound |
             glycan | reaction | rclass | enzyme | network | variant | disease |
             drug | dgroup | disease_ja | drug_ja | dgroup_ja | compound_ja

&lt;option&gt; = aaseq | ntseq | mol | kcf | image | conf | kgml | json</pre>

<b class="txt3">Description</b>
<div class="indent2">
This operation retrieves given database entries in a flat file format or
 in other formats with options. Flat file formats are available for all 
KEGG databases except brite. The input is limited up to 10 entries.
<br><br>
Options allow retrieval of selected fields, including sequence data from
 genes entries, chemical structure data or gif image files from 
compound, glycan and drug entries, png image files or kgml files from 
pathway entries. The input is limited to one compound/glycan/drug entry 
with the image option, and to one pathway entry with the image or kgml 
option.
</div>

<b class="txt3">Examples</b>
<div class="indent2">
<table>
<tbody><tr>
  <td><a href="https://rest.kegg.jp/get/C01290+G00092">/get/C01290+G00092</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>retrieves a compound entry and a glycan entry</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/get/hsa:10458+ece:Z5100">/get/hsa:10458+ece:Z5100</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>retrieves a human gene entry and an E.coli O157 gene entry</td>
</tr>
<tr class="top">
  <td><a href="https://rest.kegg.jp/get/hsa:10458+ece:Z5100/aaseq">/get/hsa:10458+ece:Z5100/aaseq</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>retrieves amino acid sequences of a human gene and an E.coli O157 gene</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/get/C00002/image">/get/C00002/image</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>retrieves the gif image file of a compound</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/get/hsa00600/image">/get/hsa00600/image</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>retrieves the png image file of a pathway map</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/get/map00600/image2x">/get/map00600/image2x</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>retrieves the doubled-sized png image file of a reference pathway map <span class="new">New!</span></td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/get/hsa00600/conf">/get/hsa00600/conf</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>retrieves the conf file of a pathway map</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/get/hsa00600/kgml">/get/hsa00600/kgml</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>retrieves the kgml file of a pathway map</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/get/br:br08301">/get/br:br08301</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>retrieves the htext file of a brite hierarchy</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/get/br:br08301/json">/get/br:br08301/json</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>retrieves the json file of a brite hierarchy</td>
</tr>
</tbody></table>
</div>

<b class="txt3">See also</b>
<div class="indent2">
<a href="https://www.kegg.jp/kegg/rest/weblink.html">KEGG WebLinks</a> to retrieve entries in HTML.
</div>

</div>

<br>
<hr class="frame3">

<h4 class="txt3" id="conv">CONV</h4>

<div class="indent">

<b class="txt3">Name</b>
<div class="indent2">
conv – convert KEGG identifiers to/from outside identifiers
</div>

<b class="txt3">URL form</b>
<pre class="block">https://rest.kegg.jp/conv/&lt;target_db&gt;/&lt;source_db&gt;

(&lt;target_db&gt; &lt;source_db&gt;) = (&lt;kegg_db&gt; &lt;outside_db&gt;) | (&lt;outside_db&gt; &lt;kegg_db&gt;)

For gene identifiers:
&lt;kegg_db&gt; = &lt;org&gt;
&lt;org&gt; = <i>KEGG organism code or T number</i>
&lt;outside_db&gt; = ncbi-geneid | ncbi-proteinid | uniprot

For chemical substance identifiers:
&lt;kegg_db&gt; = compound | glycan | drug
&lt;outside_db&gt; = pubchem | chebi</pre>

<pre class="block">https://rest.kegg.jp/conv/&lt;target_db&gt;/&lt;dbentries&gt;

For gene identifiers:
&lt;dbentries&gt; = <i>database entries of the following</i> &lt;database&gt;
&lt;database&gt; = &lt;org&gt; | genes | ncbi-geneid | ncbi-proteinid | uniprot
&lt;org&gt; = <i>KEGG organism code or T number</i>

For chemical substance identifiers:
&lt;dbentries&gt; = <i>database entries of the following</i> &lt;database&gt;
&lt;database&gt; = compound | glycan | drug | pubchem | chebi</pre>

<b class="txt3">Description</b>
<div class="indent2">
This operation can be used to convert entry identifiers (accession 
numbers) of outside databases to KEGG identifiers, and vice versa. The 
first form allows database to database mapping, while the second form 
allows conversion of a selected number of entries. The database name 
"genes" may be used only in the second form.
</div>

<b class="txt3">Examples</b>
<div class="indent2">
<table>
<tbody><tr>
  <td><a href="https://rest.kegg.jp/conv/eco/ncbi-geneid">/conv/eco/ncbi-geneid</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>conversion from NCBI GeneID to KEGG ID for E. coli genes</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/conv/ncbi-geneid/eco">/conv/ncbi-geneid/eco</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>opposite direction</td>
</tr>
<tr>
  <td class="nowrap"><a href="https://rest.kegg.jp/conv/ncbi-proteinid/hsa:10458+ece:Z5100">/conv/ncbi-proteinid/hsa:10458+ece:Z5100</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>conversion from KEGG ID to NCBI ProteinID</td>
</tr>
<tr class="top">
  <td><a href="https://rest.kegg.jp/conv/genes/ncbi-geneid:948364">/conv/genes/ncbi-geneid:948364</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>conversion from NCBI GeneID to KEGG ID when the organism code is not known</td>
</tr>
</tbody></table>
</div>

<b class="txt3">See also</b>
<div class="indent2">
<a href="https://www.kegg.jp/kegg/tool/conv_id.html">Convert ID</a> tool in <a href="https://www.kegg.jp/kegg/mapper.html">KEGG Mapper</a>.
</div>

</div>

<br>
<hr class="frame3">

<h4 class="txt3" id="link">LINK</h4>

<div class="indent">

<b class="txt3">Name</b>
<div class="indent2">
link – find related entries by using database cross-references
</div>

<b class="txt3">URL form</b>
<pre class="block">https://rest.kegg.jp/link/&lt;target_db&gt;/&lt;source_db&gt;

&lt;target_db&gt; = &lt;database&gt;
&lt;source_db&gt; = &lt;database&gt;

&lt;database&gt; = pathway | brite | module | ko | &lt;org&gt; | vg | vp | ag | genome | compound |
             glycan | reaction | rclass | enzyme | network | variant | disease |
             drug | dgroup | &lt;outside_db&gt;
&lt;outside_db&gt; = pubmed | atc | jtc</pre>

<pre class="block">https://rest.kegg.jp/link/&lt;target_db&gt;/&lt;dbentries&gt;

&lt;dbentries&gt; = <i>KEGG database entries of the following</i> &lt;database&gt;
&lt;database&gt; = pathway | brite | module | ko | &lt;org&gt; | vg | vp | ag | genome | compound |
             glycan | reaction | rclass | enzyme | network | variant | disease |
             drug | dgroup | &lt;outside_db&gt;
&lt;outside_db&gt; = pubmed | atc | jtc</pre>

<b class="txt3">Description</b>
<div class="indent2">
This operation allows retrieval of cross-references within all KEGG 
databases, as well as between KEGG databases and outside databases. It 
is useful for finding various relationships, such as relationships 
between genes and pathways. The first form allows retrieval of database 
to database cross-references, while the second form allows retrieval for
 a selected number of entries. The database name "genes" may be used 
only for "ko" entries in the second form.
</div>

<b class="txt3">Examples</b>
<div class="indent2">
<table>
<tbody><tr>
  <td><a href="https://rest.kegg.jp/link/pathway/hsa">/link/pathway/hsa</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>KEGG pathways linked from each of the human genes</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/link/hsa/pathway">/link/hsa/pathway</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>human genes linked from each of the KEGG pathways</td>
</tr>
<tr class="top">
  <td><a href="https://rest.kegg.jp/link/pathway/hsa:10458+ece:Z5100">/link/pathway/hsa:10458+ece:Z5100</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>KEGG pathways linked from a human gene and an E. coli O157 gene</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/link/genes/K00500">/link/genes/K00500</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>List of genes with the KO assignment of K00500</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/link/hsa/hsa00010">/link/hsa/hsa00010</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>List of human genes in pathway hsa00010</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/link/ko/map00010">/link/ko/map00010</a>
  or
  <a href="https://rest.kegg.jp/link/ko/ko00010">/link/ko/ko00010</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>List of KO entries in pathway map00010 or ko00010</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/link/rn/map00010">/link/rn/map00010</a>
  or
  <a href="https://rest.kegg.jp/link/rn/rn00010">/link/rn/rn00010</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>List of reaction entries in pathway map00010 or rn00010</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/link/ec/map00010">/link/ec/map00010</a>
  or
  <a href="https://rest.kegg.jp/link/ec/ec00010">/link/ec/ec00010</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>List of EC number entries in pathway map00010 or ec00010</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/link/cpd/map00010">/link/cpd/map00010</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>List of compound entries in pathway map00010</td>
</tr>
</tbody></table>
</div>

</div>

<br>
<hr class="frame3">

<h4 class="txt3" id="ddi">DDI</h4>

<div class="indent">

<b class="txt3">Name</b>
<div class="indent2">
ddi – find adverse drug-drug interactions
</div>

<b class="txt3">URL form</b>
<pre class="block">https://rest.kegg.jp/ddi/&lt;dbentry&gt;

&lt;dbentry&gt; = <i>Single entry of the following</i> &lt;database&gt;
&lt;database&gt; = drug | ndc | yj</pre>

<pre class="block">https://rest.kegg.jp/ddi/&lt;dbentries&gt;

&lt;dbentries&gt; = <i>Multiple entries in one of the following</i> &lt;database&gt;
&lt;database&gt; = drug | ndc | yj</pre>

<b class="txt3">Description</b>
<div class="indent2">
This operation searches against the KEGG drug interaction database, 
where drug-drug interactions designated as contraindication (CI) and 
precaution (P) in Japanese drug labels are extracted, standardized by 
KEGG identifiers and annotated with any possible molecular mechanims. 
The first form reports all known interactions, while the second form can
 be used to check if any drug pair in a given set of drugs is CI or P.
</div>

<b class="txt3">Examples</b>
<div class="indent2">
<table>
<tbody><tr>
  <td><a href="https://rest.kegg.jp/ddi/D00564">/ddi/D00564</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>drugs that are known to interact with a given drug</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/ddi/D00564+D00100+D00109">/ddi/D00564+D00100+D00109
</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>check if drug-drug interactions are present among given drugs</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/ddi/ndc:0078-0401">/ddi/ndc:0078-0401</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>drug products that are known to interact with Gleevec</td>
</tr>
</tbody></table>
</div>

<b class="txt3">See also</b>
<div class="indent2">
<a href="https://www.kegg.jp/medicus-bin/select_drug?lang=en">Drug Interaction Checker</a> tool in <a href="https://www.kegg.jp/kegg/medicus.html">KEGG MEDICUS</a>.
</div>

</div>

<br>
<hr class="frame3">
<hr class="frame3">

<h4 class="txt3" id="link_rdf">LINK (with RDF option)</h4>

<div class="indent">

<b class="txt3">Name</b>
<div class="indent2">
link – find related entries by using database cross-references
</div>

<b class="txt3">URL form</b>
<pre class="block">https://rest.kegg.jp/link/&lt;target_db&gt;/&lt;source_db&gt;[/&lt;option&gt;]

&lt;target_db&gt; = &lt;database&gt;
&lt;source_db&gt; = &lt;database&gt;

&lt;database&gt; = drug | atc | jtc

&lt;option&gt; = turtle | n-triple</pre>

<pre class="block">https://rest.kegg.jp/link/&lt;target_db&gt;/&lt;dbentries&gt;[/&lt;option&gt;]

&lt;dbentries&gt; = <i>KEGG database entries of the following</i> &lt;database&gt;
&lt;database&gt; = drug | atc | jtc

&lt;option&gt; = turtle | n-triple</pre>

<b class="txt3">Description</b>
<div class="indent2">
This operation allows retrieval of cross-references within a few 
selected databases. The first form allows retrieval of database to 
database cross-references, while the second form allows retrieval for a 
selected number of entries.
</div>

<b class="txt3">Examples</b>
<div class="indent2">
<table>
<tbody><tr>
  <td><a href="https://rest.kegg.jp/link/atc/D01441/n-triple">/link/atc/D01441/n-triple</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>ATC code of a given KEGG DRUG entry in N-Triples format</td>
</tr>
<tr>
  <td><a href="https://rest.kegg.jp/link/jtc/D01441/turtle">/link/jtc/D01441/turtle</a></td>
  <td>&nbsp;&nbsp;</td>
  <td>Japanese therapeutic category of a given KEGG DRUG entry in Turtle format</td>
</tr>
</tbody></table>
</div>

</div>

<br>
<hr class="frame3">
<i>Last updated: May 26, 2023</i>

<br><br><a href="javascript:history.back()">« Back</a>
</div>



</body><div id="immersive-translate-popup" style="all: initial"></div></html>